components/raftstore/src/store/config.rs

```diff
@@ -264,6 +264,8 @@ pub struct Config {
     // Interval to inspect the latency of raftstore for slow store detection.
     pub inspect_interval: ReadableDuration,
 
+    pub max_batch_size: usize,
```

Comment:
What is this for?

Reply:
In our bench tests, sometimes it's useful to set larger

Reply:
Put it in another PR then, that gives a cleaner commit history.

Reply:
Ok, working on it